ag_data/formulas/formulas.py

```diff
@@ -0,0 +1,27 @@
+def identity(**kwargs):
+    return kwargs
+
+
+simple_temperature_sensor = identity
+
+
+def dual_temperature_sensor(internal, external):
+    return {"mean": (internal + external) / 2, "diff": internal - external}
+
+
+def flow_sensor(prevGasLevel, prevTimestamp, volumetricFlow, timestamp):
+    if prevGasLevel is not None and prevTimestamp is not None:
+        time_elapsed = timestamp - prevTimestamp
+        return {
+            "gasLevel": prevGasLevel - volumetricFlow * time_elapsed.total_seconds()
+        }
+    else:
+        return {"gasLevel": 100}
+
+
+formula_map = {
+    0: identity,
```

Comment:
I remember we talked about the numbering of formulas. Is it intentional to keep the even numbers, or we will have this just for now?

Reply:
I didn't want break things. That's why I kept these numbers (0,2,4,6). And it doesn't mean I endorse using these numbers. String would be better option in my opinion.

I think there are already already tons of changes in this single PR and it would be better doing this later sprint. Once we make a decision, implementation won't be hard. Total less than 10-20 lines of changes.

Reply:
sure, I'm okay if we do this later.